docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -4,50 +4,44 @@
 <titleabbrev>Add data</titleabbrev>
 ++++
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data is to use one of our many integrations.
+Alternatively, you add a sample data set or upload a file.
 
-[role="screenshot"]
-image::images/addData_home_7.15.0.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
 
 [float]
 [[add-data-tutorial-kibana]]
-=== Add data
+=== Add an integration
 
-Want to ingest logs, metrics, security, or application data?
-Install and configure a Beats data shipper or other module to periodically collect the data
-and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
+Integrations periodically collect logs, metrics, security,
+and other application data and send it to {es}.
+Each integration provides dashboards and visualizations, so you can quickly get insights into your data.
 
-[role="screenshot"]
-image::images/add-data-tutorials.png[Add Data tutorials]
+{kib} automatically guides you to add an integration when it doesn't detect any data.
+Alternatively, follow these steps.
 
-[discrete]
-=== Add Elastic Agent
+. On the home page, click *Add integrations*.
 
-*Elastic Agent* is the next generation of
-data integration modules, offering
-a centralized way to set up your integrations.
+. Select the integration for the type of data that you want to collect.
++
+[role="screenshot"]
+image::images/add-integration.png[Integrations page from which you can choose integrations to start collecting and analyzing data]
 
-With *Fleet*, you can add
-and manage integrations for popular services and platforms, providing
-an easy way to collect your data, and manage {elastic-agent} installations in standalone or {fleet} mode. The integrations
-ship with dashboards and visualizations,
-so you can quickly get insights into your data, and {fleet} mode offers several advantages:
+NOTE: {kib} uses *Elastic Agent* to ingest data. For more information,
```

Comment:
This statement is misleading because the new integrations view shows both Elastic Agent _and_ Beats integrations, depending on what the user has selected here:

![image](https://user-images.githubusercontent.com/14206422/140845153-c2d17f2e-eab9-4ebf-900e-417d513de04e.png)

I think you need to mention these options because they might not be visible until the user scrolls down. You might also want to briefly introduce the two options.